avocado/plugins/run.py

```diff
@@ -77,7 +77,10 @@ def configure(self, parser):
                                   'You can also use suffixes, like: '
                                   ' s (seconds), m (minutes), h (hours). '))
 
-        parser.add_argument('--failfast', action='store_true',
+        parser.add_argument('--failfast',
+                            nargs='?',
+                            const=True,
+                            type=self._failfast,
                             help='Interrupt job on first failed test.')
```

Comment:
Actually I thought about it more and we do use `on|off` values already, so in the end I'd probably just require the parameter as making it optional is quite strange (at least to me). Alternatively if you can come up with a nice help message which would sumarize that:

* default is disabled
* when `--failfast` is used, failfast is enabled
* the same with `--failfast on`
* the `--failfast off` force disables the failfast (for replay purposes)

then we can keep it.